examples/SampleUtils.cpp

```diff
@@ -45,10 +45,15 @@ static CmdBufType cmdBufType = CmdBufType::Terrible;
 #else
 static CmdBufType cmdBufType = CmdBufType::None;
 #endif  // defined(WEBNN_ENABLE_WIRE)
-static webnn::wire::WireServer* wireServer = nullptr;
-static webnn::wire::WireClient* wireClient = nullptr;
-static utils::TerribleCommandBuffer* c2sBuf = nullptr;
-static utils::TerribleCommandBuffer* s2cBuf = nullptr;
+
+class WireHelper {
+    public:
+    std::unique_ptr<webnn::wire::WireServer> wireServer = nullptr;
```

Comment:
nit: `unique_ptr` is already initialized to `nullptr` here, remove ` = nullptr`.

Reply:
Thanks! Updated, PTAL again. @bbernhar